### DISCUSSION TITLE
Username Policy

### DISCUSSION THREAD

**Discussion by xylonxy** [community/community#136312] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

I'm enquiring about whether username “xylon” can be released? The user who has registered that username doesn't look to have any activity. Given that I own the domain “xylon.love”, I would appreciate if you could let me know if it would be possible to release this username for my use.

Thank you for your assistance.

Regards, Xylon

**Comment by davevad93** [comment]
Hi @xylon888 ,

Take a look at the [GitHub's Username Policy](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy), regarding a user's apparent inactivity it says the following:

- ***Keep in mind that not all activity on GitHub is publicly visible; accounts with no visible activity may be in active use.***

    ***If the username you want has already been claimed, consider other names or unique variations. Using a number, hyphen, or an alternative spelling might help you identify a desirable username that's still available.***

On the other hand, If your issue concerns a trademark policy violation, you can report it to GitHub via [Submit a Trademark Policy Violation Report](https://support.github.com/contact/trademark-policy).

**Comment by ebndev** [comment]
Hi @xylon888,

We are no longer accepting requests to release dormant usernames. We are not able to accommodate this request.

If you hold a trademark to this name and believe there is a clear case of confusion between the account and your trademark, please review our [Trademark Policy](https://docs.github.com/articles/github-trademark-policy/) to determine if you should submit a trademark policy violation report.

Thank you!
